RUMM-924 Add Data Scrubbing and GDPR docs (#377)
DataDog/dd-sdk-ios: docs/trace_collection.md

    {{% /tab %}}
    {{< /tabs >}}

    To be compliant with the GDPR regulation, the SDK requires the `trackingConsent` value at initialization.
    The `trackingConsent` can be one of the following values:

    - `.pending` - the SDK starts collecting and batching the data but does not send it to Datadog. The SDK waits for the new tracking consent value to decide what to do with the batched data.
    - `.granted` - the SDK starts collecting the data and sends it to Datadog.

>     - `.notGranted` - the SDK does not collect any data: logs, traces and RUM events will not be send to Datadog.

Reviewer: ```suggestion
    - `.notGranted` - the SDK does not collect any data: logs, traces, and RUM events are not sent to Datadog.
```

>     To change the tracking consent value after the SDK is initialized, use `Datadog.set(trackingConsent:)` API.

Reviewer: ```suggestion
    To change the tracking consent value after the SDK is initialized, use the `Datadog.set(trackingConsent:)` API call.
```

>     The SDK will change its behavior according to the new value, e.g. if the current tracking consent is `.pending`:

Reviewer: ```suggestion
    The SDK changes its behavior according to the new value. For example, if the current tracking consent is `.pending`:
```
